lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -63,7 +74,10 @@ def call(params)
           symbolized_config[:labels] = symbolized_config[:labels].map { |l| GoodData::Helpers.symbolize_keys(l) }
           headers_in_options = params.csv_headers == 'false' || true
 
-          mode = params.sync_mode || 'sync_project'
+          mode = params.sync_mode
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)